Tests/SonarScanner.MSBuild.TFS.Test/ProgramTests.cs

```diff
@@ -59,29 +59,32 @@ public void Execute_ShouldExecute_CoverageConverter_ShouldSucceeed()
         {
             var logger = new TestLogger();
             var tempDir = Directory.CreateDirectory(Path.Combine(Path.GetTempPath(), Guid.NewGuid().ToString())).FullName;
-            var text = @"<?xml version=""1.0"" encoding=""utf-8""?>
-                        <AnalysisConfig xmlns:xsd=""http://www.w3.org/2001/XMLSchema""
-                        xmlns:xsi=""http://www.w3.org/2001/XMLSchema-instance"" xmlns=""http://www.sonarsource.com/msbuild/integration/2015/1"">
-                        </AnalysisConfig>";
+            // Because the test is run now with VS community edition installed, the nuget package is used and the path to the .exe has to be provided.
```

Comment:
```suggestion
            // Because the test is run now with VS community edition installed, the "Microsoft.CodeCoverage" NuGet package is used and the path to the .exe has to be provided.
```